### DISCUSSION TITLE
Meta: Community cleanup effort for Issues

### DISCUSSION THREAD

**Discussion by voronoipotato** [dotnet/fsharp#9946] [General]
This issue is to track easy to pick up issues, and outdated issues managed by (and for) the community. 

@abelbraaksma  and I were talking in our most recent dugnad about one of the challenges in contributing to F# and we felt that combing through the issues was a bit challenging. As a community we can work to try to help reduce the load on the F# team, and share responsibility of the language we love. Primarily this issue is for listing issues we think are ready to be worked and perhaps somewhat easier, suitable for dugnads, however this issue may also be used for tracking and discussing potential duplicate issues or black hole issues. 

@dsyme  @cartermp I added you both to this issue because I feel you would be best equipped to guiding us in lightening the load for you.

**Comment by voronoipotato** [comment]
Vlad in the #dugnad [slack](http://fsharp.slack.com), which you can get invited to through [fsharp.org](http://fsharp.org) suggested that issues under [Testing Overhaul](https://github.com/dotnet/fsharp/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Testing+Overhaul%22) are often good to pick up.

**Comment by cartermp** [comment]
If you go through the earliest reports, a good lot of them are legit bugs that have existed for a long time and don't need to be revisited.

Doing a pass of tooling-related issues (usually tagged `area-FCS`, `area-IDE Language Service`, `area-IDE Debug`, `area-IDE FSI`) there are sometimes issues that are not really a problem anymore.

Whenever there is a PR related to a feature or tooling area (e.g., tooltips), it's good to do a pass over all issues with a relevant keyword mentioned and see if that PR resolves the issue. I've done this a few times and been able to close out a few bugs.

Some issues are kind of confusing because the actual reported problem is fixed, but a follow-up is present and it wasn't transferred to a new issue. These should be re-triaged.

One thing I'll mention is that I did a lot of issue-scavenging a few years back and closed out the majority of fixed ones, or where the behavior mentioned is no longer relevant. So it may be hard to come by issues that aren't a legit problem.

**Comment by abelbraaksma** [comment]
A lot of clean up has been, and is still taking place, which is great! This includes tagging all non tagged issues, and close what can be closed, I really like seeing all the comments flow in :). 

Once this "storm" is over, I propose to revisit this, and perhaps have a look at listing "up for grabs" stuff (some repos do that in the contributing.md), and maybe other things as well.

**Comment by cartermp** [comment]
Yes, there also need to be:

* Some level of impact assigned to feature requests/improvements
* A tag applied to anything we think is a good first issue for contributors
